wsmaster/che-core-api-factory/src/main/java/org/eclipse/che/api/factory/server/scm/AuthorizingFileContentProvider.java

```diff
@@ -80,7 +80,7 @@ private String fetchContent(
         String authorization;
         if (isNullOrEmpty(credentials)) {
           PersonalAccessToken token =
-              personalAccessTokenManager.get(remoteFactoryUrl.getHostName());
+              personalAccessTokenManager.get(remoteFactoryUrl.getProviderUrl());
```

Comment:
Will it affect existed PATs?

Reply:
It won't. This is just a refactoring of a method which retrieves SCM endpoint. The output is the same but the method is different.